edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -346,6 +347,9 @@ impl MakeModuleRuntime for DockerModuleRuntime {
                             system_resources: Arc::new(Mutex::new(system_resources)),
                             notary_registries,
                             notary_lock,
+                            allow_privileged_docker_containers: settings
```

Comment:
I don't see allow_privileged_docker_containers defined above? Where does it come from?

Reply:
`allow_privileged_docker_containers` is the field being set, just like system_resources. The other fields are taking advantage of the rust shorthand that allows `notary_lock: notary_lock` to be simplified to just `notary_lock`.